profiler/README.md

```diff
@@ -13,3 +13,4 @@ It should be considered experimental and is completely unsupported.
 |`splunk.profiler.keep-files`         | false                  | leave JFR files on disk id `true`         |
 |`splunk.profiler.logs-endpoint`      | http://localhost:4317  | where to send OTLP logs                   |
 |`splunk.profiler.period.{eventName}` | n/a                    | customize period (in ms) for a specific jfr event. For example, to set the ThreadDump frequency to 1s (100ms): `-Dsplunk.profiler.period.threaddump=1000` |
+|`splunk.profiler.tlab.enabled`       | true                   | set to `false` to disable TLAB memory events |
```

Comment:
don't we want this off by default, due to the cost?

Reply:
I think this is still an open question. We haven't yet really quantified the cost yet, so maybe to play it safe is a better idea. What do @iNikem and @mateuszrzeszutek and @laurit think?

Reply:
My vote would be to leave it off by default until we've at least verified it's not going to destroy the average app's performance.

Reply:
It's still turned off unless `splunk.profiler.enabled=true`, which is not the default (yet?).

Reply:
Ok, I think you're right and that this is a better idea, so I've inverted the logic around this and defaulted tlab support to off.